### PR TITLE
Feature/integrations bigcommerce

### PR DESCRIPTION
**Pull request by Aduralere54256703**
corrections made to this branch

### CODE REVIEW

**Review by harlick-b — DISMISSED** [review]
ok

File: assets/images/integrations/bigcommerce/Reinstall.gif
Comment (harlick-b) [inline assets/images/integrations/bigcommerce/Reinstall.gif:1]: Image name should start with a lowercase character

**Review by harlick-b — COMMENTED** [review]
Check comment

**Review by thetallralph — CHANGES_REQUESTED** [review]
@Aduralere54256703 
I made some quick change:
- Updated the integration URL to reflect our new domain (from bns.re to trybeans.com)
- Reformatted the Order Logs section to use bullet points for better scanability

But in order to have more consistency with other platform article, you should  add a new "Rewards Page" section.

File: assets/images/integrations/bigcommerce/Reinstall.gif
Comment (thetallralph) [reply assets/images/integrations/bigcommerce/Reinstall.gif:1]: ok

**Review by thetallralph — APPROVED** [review]
ok

**Pull request merged**